pkg/parser/expression.go

```diff
@@ -122,6 +124,14 @@ func (p *parser) isAtExprEnd() bool {
 	return p.isAtEOL()
 }
 
+func (p *parser) parseIdentExpr() Node {
+	funcDef := p.funcs[p.cur.Literal]
+	if funcDef != nil && funcDef.isNiladic() {
```

Comment:
In `parseTopLevelExpr()` you don't bother with the `funcDef` var - you just index `p.funcs[]` twice. I suggest you do the same here or use the var over there.

Reply:
hmm, I tried this. Above I did it because it allowed me drop an extra nested if statement, which i think reads clearer, but hear I feel the local var reads clearer 
as it is not conditioned on `tok.Type == lexer.IDENT` - what do you think?

Reply:
All good. Keep as is.